TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/timeline/ProfileTimeline.java

```diff
@@ -0,0 +1,155 @@
+package com.ybsystem.tweetmate.fragments.timeline;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.Fragment;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.adapters.recycler.TweetRecyclerAdapter;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.models.entities.Column;
+import com.ybsystem.tweetmate.models.entities.twitter.TwitterStatus;
+import com.ybsystem.tweetmate.utils.ExceptionUtils;
+import com.ybsystem.tweetmate.utils.ToastUtils;
+
+import java.util.List;
+
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers;
+import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.observers.DisposableObserver;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import twitter4j.Paging;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+import static com.ybsystem.tweetmate.models.enums.ColumnType.*;
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class ProfileTimeline extends TimelineBase {
+
+    // Bundle
+    private Column mColumn;
+    private boolean mIsPrivate;
+
+    public Fragment newInstance(Column column, boolean isPrivate) {
+        Bundle bundle = new Bundle();
+        bundle.putSerializable("COLUMN", column);
+        bundle.putSerializable("PRIVATE", isPrivate);
+        setArguments(bundle);
+        return this;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate
+        View view = inflater.inflate(R.layout.fragment_timeline, container, false);
+
+        // Init
+        mColumn = (Column) getArguments().getSerializable("COLUMN");
+        mIsPrivate = (boolean) getArguments().getSerializable("PRIVATE");
+        mScrollLoad = true;
+
+        // Set view
+        setRecyclerView(view);
+        setRecyclerAdapter(new TweetRecyclerAdapter());
+        setUltraPullToRefresh(view);
+        mPtrFrame.setEnabled(false);
+
+        return view;
+    }
+
+    protected void loadTweet(boolean isPullLoad, boolean isClickLoad) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 29 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS3776)